wdl-lint/tests/lints/disallowed-declaration-name/source.errors

```diff
@@ -0,0 +1,76 @@
+note[MisplacedLintDirective]: lint directive `InputSorting` has no effect above task definition
```

Comment:
Fix this by putting the directive in the right location (sorry for suggesting the wrong one!)